### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by sanu29**
image

### CODE REVIEW

File: src/component/AllComponnts.jsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export {Footer} from "./Footer"
```
Comment (iRohitGaur) [inline src/component/AllComponnts.jsx]: suggestion: `AllComponents` file name is misspelled

File: src/component/HomepageBody.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+const {categories} = useCategoriesContext();
+if(categories === 'loading')
+{
+return (<h1>
+  <ThreeDots color="#2BAD60" height="100" width="100" />
+</h1>
+)
+}
+else{
```
Comment (iRohitGaur) [inline src/component/HomepageBody.jsx]: suggestion: some issue with indentation. Please look into it

File: src/component/HomepageBody.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      return(
+      <Link to={'/product'} className="card border-radius-sm">
+      <img className="card-img-main border-radius-sm " src={item.image} alt="jacket" />
+      <div className="card-primary">
+        <div className="card-subtitle">{item.categoryName}</div>
+        <h2 className="card-title"><span className="text-muted">Upto </span> 10% Off</h2>
+      </div>
+      </Link>
+      )
+      }
```
Comment (iRohitGaur) [inline src/component/HomepageBody.jsx]: suggestion: indentation issue. use extension like prettier

File: src/context/categories-context.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            }
+            catch(err)
+            {
+                console.log(err)
+            }
+         }
+         getCategories()
+     
+    },[])
+    // console.log(categories)
```
Comment (iRohitGaur) [inline src/context/categories-context.jsx]: suggestion: remove commented code

File: src/context/categories-context.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    const [categories , setCategories] = useState()
+    useEffect (() =>{
+         async function getCategories(){
+            try{
+            const response =  await ( axios.get( '/api/categories'))
+             setCategories(response.data.categories)
+            }
+            catch(err)
+            {
+                console.log(err)
```
Comment (iRohitGaur) [inline src/context/categories-context.jsx]: suggestion: remove `console.log`

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Code is easy to understand.

Good to merge 👍🏻

File: src/App.css [hunk 6]
```diff
@@ -1,51 +1,51 @@
...
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.bg-primary {
+    background-color: var(--primary);
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
```
Comment (sanjitsarkar) [inline src/App.css]: **Suggestion:** Prefer proper naming conventions for CSS classes.
Always use uppercase for naming classes.

File: src/component/HomepageBody.jsx [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+      })}
+
+
+    </div>
+  </div>
+
```
Comment (zabihshaik) [inline src/component/HomepageBody.jsx:73]: Better to remove unnecessary blank spaces

File: src/component/HomepageBody.jsx [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+
+  <div className="margin-top d-flex-column justify-content-center align-items-center">
+
+    <h1 className="heading2 margin-bottom text-primary">Our Features</h1>
+
+    <div className="d-flex justify-content-center align-items-center responsive-image flex-wrap ">
+
+      {<img src="https://instantgrocery.netlify.app/assets/images/feat.png" className="width-100" alt="" srcset="" /> }
```
Comment (zabihshaik) [inline src/component/HomepageBody.jsx]: Better to add meaningful text in alt of img

**Review by sanjitsarkar — APPROVED** [review]

File: src/context/categories-context.jsx [hunk 9]
```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { createContext, useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
```
Comment (zabihshaik) [inline src/context/categories-context.jsx]: can have these imports in line number 2 itself

**Review by zabihshaik — APPROVED** [review]
Everything looks good. Please check the suggestions in comments.

File: src/component/AllComponnts.jsx (hunk 1 above)
Comment (sanu29) [reply src/component/AllComponnts.jsx]: Changes Done

File: src/component/HomepageBody.jsx (hunk 2 above)
Comment (sanu29) [reply src/component/HomepageBody.jsx]: Changes Done

File: src/component/HomepageBody.jsx (hunk 3 above)
Comment (sanu29) [reply src/component/HomepageBody.jsx]: Changes Done

File: src/component/HomepageBody.jsx (hunk 8 above)
Comment (sanu29) [reply src/component/HomepageBody.jsx]: Changes Done

File: src/context/categories-context.jsx (hunk 9 above)
Comment (sanu29) [reply src/context/categories-context.jsx]: Changes Done

**Pull request merged**